scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementSample.java

```diff
@@ -100,43 +108,55 @@ public void start(Stage stage) {
       GraphicsOverlay absoluteOverlay = new GraphicsOverlay();
       absoluteOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.ABSOLUTE);
 
-      // create point for graphic location
-      Point point = new Point(-4.04, 53.06, 1000, SpatialReferences.getWgs84());
+      GraphicsOverlay relativeToSceneOverlay = new GraphicsOverlay();
+      relativeToSceneOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.RELATIVE_TO_SCENE);
+
+      // create points for graphic locations
+      Point surfaceRelatedPoint = new Point(-4.4609257, 48.3903965, 70, SpatialReferences.getWgs84());
+      Point sceneRelatedPoint = new Point(-4.4610562, 48.3902727, 70, SpatialReferences.getWgs84());
```

Comment:
Although these var names might be more accurate (I don't actuall know!), the rest of the vars in this sample have a slightly different naming convention. So perhaps we should update these to match the rest (e.g.`relativeToSceneOverlay`). I'll leave the final word here to @Rachael-E , she's the nomenclature expert.

```suggestion
      Point relativeToSurfacePoint = new Point(-4.4609257, 48.3903965, 70, SpatialReferences.getWgs84());
      Point relativeToScenePoint = new Point(-4.4610562, 48.3902727, 70, SpatialReferences.getWgs84());
```

Reply:
Have left this for Rachael to comment when she reviews.

Reply:
sorry @jenmerritt I just saw this. Let's go with Jon's suggestion, it makes it more clear what the point is for.